NetworkLayer/Model/DataRepresentation/AdvertsListElement.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AdvertsListElement.swift
+//  NetworkLayer
+//
+//  Created by Clément Garcia on 17/10/2022.
+//
+
+import Foundation
+
+/// Descibe an element of the adverts list
+struct AdvertsListElement: Codable {
+    let id: Int
+    let categoryId: Int?
+    let title: String
+    let description: String?
```

Comment:
```suggestion
    let description: String
```
Je pense que tu peux retirer l'optionnel ici